dev-haskell/gtk/gtk-0.15.9.ebuild

```diff
@@ -13,28 +13,23 @@ DESCRIPTION="Binding to the Gtk+ graphical user interface library"
 HOMEPAGE="https://projects.haskell.org/gtk2hs/"
 
 LICENSE="LGPL-2.1"
-SLOT="2/${PV}" # check SLOT when bumping!
+SLOT="0/${PV}"
```

Comment:
You'll need to do something extra to change the slot (I agree it should be changed, though).

https://devmanual.gentoo.org/ebuild-maintenance/package-moves/index.html#changing-ebuilds-slot